### DISCUSSION TITLE
Cycleway overlay

### DISCUSSION THREAD

**Discussion by gerion0** [organicmaps/organicmaps#483] [Ideas]
Many thanks for the app and bringing it to F-Droid. It is simple to use, just works and especially renders on older phones!

I often make bike trips. For that, in Germany (and several other, mostly European, states) official bike routes exists that often have a nice landscape, good quality etc. These routes are mapped in OSM. A very good online map to display them is https://cycling.waymarkedtrails.org/. This website also has a few other profiles: Hiking, Mountainbiking, etc.. The routes there are highlighted and clickable to get some extra information.

Since all data is coming from OSM, it should be possible to integrate such a functionality directly in Organic Maps, too (as offline overlay for example). For biking, this would be very helpful. Maybe you like the idea.

**Comment by cyberchrime** [comment]
Another great rendering is https://www.cyclosm.org, also based on OSM. Besides cycling routes it also renders cycling tracks, speed limited streets and POIs which are especially interesting for cyclists.

**Comment by biodranik** [comment]
It would be great to implement some ideas from these resources directly in the OM.

**Comment by fcottet** [comment]
It would be nice to be able to add any overlay, such as https://vv.fork.pl/index.php

**Reply by biodranik** [reply]
We don't want to create another OSMand.

**Comment by pastk** [comment]
Its discussed in more detail here: https://github.com/organicmaps/organicmaps/issues/1577

Official/national/regional routes highlighting will be very useful for hikers too - for that we need to add support for route relations. I think relations are taken into account for the roads already (e.g. national roads names and codes like E90 are shown in OM).
